antarest/study/storage/auto_archive_service.py

```diff
@@ -84,10 +84,16 @@ def _try_archive_studies(self) -> None:
                     exc_info=e,
                 )
 
+    def _try_clear_snapshot(self) -> None:
+        self.study_service.storage_service.variant_study_service.clear_all_snapshots(
+            datetime.timedelta(days=self.config.storage.snapshot_retention_days)
+        )
+
     def _loop(self) -> None:
         while True:
             try:
                 self._try_archive_studies()
+                self._try_clear_snapshot()
```

Comment:
Since you've added this you can change the code in the same file line 45 to 78 to only check for raw studies as variants will be cleaned way more often with your method. And it will simplify the code

Reply:
Nevermind we have to keep this to archive outputs